build.gradle

```diff
@@ -130,7 +130,8 @@ repositories {
   mavenLocal()
   mavenCentral()
   maven {
-    url 'https://jitpack.io'
+    name = "AzureArtifacts"
+    url = uri("https://pkgs.dev.azure.com/hmcts/Artifacts/_packaging/hmcts-lib/maven/v1")
   }
 }
 
```

Comment:
reformLogging = '6.0.1'
can you please check this version, as this is not present on the 
https://dev.azure.com/hmcts/Artifacts/_artifacts/feed/hmcts-lib/maven/com.github.hmcts.java-logging%2Flogging/versions